player/public/libs/plugin-framework.min.js

```diff
@@ -765,6 +765,12 @@ org.ekstep.pluginframework.pluginManager = new(Class.extend({
         var instance = this;
         org.ekstep.pluginframework.resourceManager.loadResource(dependency.src, 'text', function(err, data) {
             if (err) {
+                EkstepRendererAPI.logErrorEvent(err, {
```

Comment:
How plugin-framework knows, EkstepRendererAPI exist?
It is an independent library. Don't put our code in plugin-framework.min.js file

Reply:
Use org.ekstep.pluginframework.eventManager.dispatchEvent

Reply:
Changes have been resolved.